python/lib/communication/setup.py

```diff
@@ -22,6 +22,6 @@
     license='',
     include_package_data=True,
     #install_requires=['websockets', 'jsonschema'],vi
-    install_requires=['dmod-core>=0.1.2', 'websockets>=8.1', 'jsonschema', 'redis'],
+    install_requires=['dmod-core>=0.1.2', 'websockets>=8.1', 'jsonschema', 'redis', 'ngen-config>=0.1.1'],
```

Comment:
For now we should probably install from `ngen_config @ git+https://github.com/noaa-owp/ngen-cal@master#egg=ngen_config&subdirectory=python/ngen_conf`. If `ngen-config` ever gets on `pypi` we should switch it however.

Reply:
I see that it is in `requirements.txt`, but you know im all ornery about being able to install things using just `pip` and `setup.py` / `setup.cfg` 😄.